### PR TITLE
Updated species list and license

### PR DESCRIPTION
**Pull request by phylojenie**
Fixed a type in the species list and added my name to the license

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -19,3 +19,7 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
```
Comment (indigotran) [inline LICENSE]: Do we all agree on using the MIT license? I know this is the base for the project, but we could look into other licenses that might fit ours better?

**Review by indigotran — COMMENTED** [review]
Also did you cite which sources you used to compile this list? You mentioned a couple of websites.

File: LICENSE (hunk 1 above)
Comment (EKROFT) [reply LICENSE]: I'm happy with MIT

**Review by EKROFT — APPROVED** [review]

**Review by Crataegus9 — APPROVED** [review]

**Review by EKROFT — APPROVED** [review]

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 DataDrivenEcologicalSynthesis
+Copyright (c) 2020 Emily Kroft, Étienne Lacroix-Carignan, Jenna LeBlanc, Amanda Liczner & Indigo Tran
```
Comment (indigotran) [inline LICENSE:3]: Are we doing alphabetically with our last name? Also Oxford comma.

Comment (Crataegus9) [reply LICENSE:3]: Yeah, I thought alphabetically was the simplest way, and it was suggested by Timothé. We can create a poll on the use of the Oxford comma, that would be fun :P

Comment (indigotran) [reply LICENSE:3]: I'm for Oxford comma 🤚

File: LICENSE (hunk 1 above)
Comment (phylojenie) [reply LICENSE]: I really don't know much about licensing, so I'm fine with MIT but if there's a better one I'm open to suggestions.

**Pull request merged**